examples/react/src/contexts/WalletSelectorContext.tsx

```diff
@@ -27,7 +27,9 @@ interface WalletSelectorContextValue {
 const WalletSelectorContext =
   React.createContext<WalletSelectorContextValue | null>(null);
 
-export const WalletSelectorContextProvider: React.FC = ({ children }) => {
+export const WalletSelectorContextProvider: React.FC<{
+  children: React.ReactNode;
```

Comment:
I don't think this is needed as `React.FC` already defines this for us. You've left the `projectId` in the code too 😛